### ISSUE TITLE
terrain doesn't load in windows/edge or ie 

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#538]
Terrain layer does not draw in windows/edge or ie. In Walkabout or any of the terrain-demos demos the other layers and styles draw but terrain is missing.

**Comment by bcamper** [comment]
Console errors?

**Comment by meetar** [comment]
Looks like it's a shader problem with all raster tiles. Console error:

```
Tangram v0.12.1 [error]: Shader compilation errors
(0, 0): Internal compiler error
```

Test scene:

```
sources:
    normals:
        type: Raster
        url: https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png?api_key=mapzen-xxxxxxx
layers:
    terrain:
        data: { source: normals }
        draw:
            raster:
                order: 0
```

**Comment by bcamper** [comment]
maybe try that shader simplification you and @dmvaldman were looking at
recently

**Comment by bcamper** [comment]
Fixed by 73517176ed593ed9bfb7b103b24f819fa409f666

IE/Edge don't allow you to instantiate an array with a coerced integer value as the size (I'm not sure if this is allowed by the spec or not, but the solution is simple enough so I'm not going to go track it down...).

So now we just make sure to output a plain stringified version of the integer.

**Comment by bcamper** [comment]
Release in [v0.12.2](https://github.com/tangrams/tangram/releases/tag/v0.12.2)
